js/services/AuthAPI.js

```diff
@@ -429,7 +429,13 @@ define(function(require, exports) {
 
 	var setAuthParams = function (tokenHeader) {
         token(tokenHeader);
-        loadUserInfo();
+        return new Promise((resolve, reject) => {
```

Comment:
Why do you need to wrap it in the separate promise? Why not just to `return loadUserInfo();`?

Reply:
Yes, it's no need to wrap. I'll fix it.